src/modules/75-cf/constants/GitSyncConstants.ts

```diff
@@ -5,11 +5,14 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-export const AUTO_COMMIT_MESSAGES = {
+export const AUTO_COMMIT_MESSAGES: {
+  [key: string]: string
+} = {
```

Comment:
You can type this as a `Record`
```suggestion
export const AUTO_COMMIT_MESSAGES: Record<string, string> = {
```